.github/workflows/cloud-integration-test.yml

```diff
@@ -0,0 +1,73 @@
+name: Cloud Integration Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      job_to_run:
+        description: 'Job to run'
+        required: true
+        default: 'cleanup_indexes'
```

Comment:
1. set the default to not cleanup_indexes
2. We consider cleanup_indexes to be an emergency measure. We should rename it and document it (perhaps to `delete_all_test_indexes` appropriately. 
3. This should delete _all_ indexes with the `test-index` prefix